openedx/core/djangoapps/courseware_api/views.py

```diff
@@ -105,6 +106,7 @@ def get_object(self):
         else:
             user_has_access = True
         overview.user_has_access = user_has_access
+        overview.user_has_staff_access = has_access(self.request.user, 'staff', overview).has_access
```

Comment:
Nit: I don't think the `has_access` attribute call is actually necessary, but I'm definitely fine with being explicit.

Reply:
Without it, I get an error saying it can't serialize an AccessResponse.  Do I maybe need to change the type of serializer?

Reply:
Oh, right. Because why `AccessResponse` is "truthy" (implements `__bool__`), it doesn't have an explicit serialization for DRF. Yup, calling `.has_access` explicitly sounds good.